design/2022.exponential-backoff.md

```diff
@@ -0,0 +1,126 @@
+### Goals
+
+- Ensure that failed certificate issuances does not overwhelm external services (to prevent putting extra strain on the external services as well as to limit hitting rate limits for cert issuance) by backing off with an exponentially increasing time period. 
+- Ensure that users can see somewhere when the issuance will be retried and that the backoff mechanism does not introduce extra complexity for debugging and fits in with the already existing 1 hour backoffs
+- Ensure that the backoff mechanism is similar to how it is implemented elsewhere on Kubernetes (i.e pod crashloopbackoff)
+- Ensure that cmctl renew can still be used to manually force renewal _now_
```

Comment:
> The goal here would be to provide an escape hatch for users, and renew would be how to reach that - is that fair to say?

Yes and also this is how `cmctl renew` works now- you can use it to retry renewal immediately without waiting for 1 hour and I have seen that it is being used in cases where the issuance failed then some external fix was made (i.e DNS setup fixed) and the user would like to see if the fix worked without waiting. In case of the delay period being even longer than an hour, it would be even more important to have this escape hatch.
I'll add a note about that 👍🏼